content/en/network_monitoring/cloud_network_monitoring/network_analytics.md

```diff
@@ -31,6 +31,8 @@ The Network Analytics page provides insights into your overall network health an
 
 To refine your search to traffic between particular endpoints, aggregate and filter your network connections **with tags**. Tags from Datadog integrations or [Unified Service Tagging][12] can be used for aggregating and filtering automatically. When utilizing tagging in Network Monitoring, you can take advantage of how network traffic flows across availability zones for a particular service or for your entire infrastructure. Grouping by `client` and `server` tags visualizes the network flow _between_ those two sets of tags.
 
+Additionally, Datadog provides a list of default [out-of-the-box](#default-tags) tags that you can use to efficiently query and analyze the network traffic most relevant to your needs.
```

Comment:
Consider omitting "efficiently," seems similar to "easily" and other subjective words we try to avoid.

Reply:
Chatted in slack and electing to leave it, as it's important for the user to know that using OOTB tags is easier than using custom tags for querying in this instance.